client_dashboard/common/src/commonMain/kotlin/org/thechance/common/domain/entity/UserTokens.kt

```diff
@@ -0,0 +1,6 @@
+package org.thechance.common.domain.entity
+
+data class UserTokens(
```

Comment:
Why did you separate this class? What happens if we add this attributes to the user class (login)
?!!